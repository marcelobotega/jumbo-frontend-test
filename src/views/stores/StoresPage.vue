<template>
  <div class="stores-page-wrapper">
    <StoresList />
  </div>
</template>

<script lang="ts">
import { Component, Vue } from "vue-property-decorator";
import StoreList from "@/components/stores/StoreList.vue";

@Component({
  components: {
    StoresList: StoreList,
  },
})
export default class StoresPage extends Vue {}
</script>
