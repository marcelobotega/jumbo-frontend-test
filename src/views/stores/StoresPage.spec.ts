import { shallowMount } from "@vue/test-utils";
import StoresPage from "@/views/stores/StoresPage.vue";

describe("StoresPage.vue", () => {
  it("should create", () => {
    const wrapper = shallowMount(StoresPage);
    expect(wrapper).toMatchSnapshot();
  });
});
