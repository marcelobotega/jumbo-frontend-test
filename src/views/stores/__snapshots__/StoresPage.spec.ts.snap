// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoresPage.vue should create 1`] = `
<div class="stores-page-wrapper">
  <storeslist-stub></storeslist-stub>
</div>
`;
